src/mono/msbuild/android/build/AndroidApp.targets

```diff
@@ -132,7 +132,6 @@
         MonoRuntimeHeaders="$(MicrosoftNetCoreAppRuntimePackNativeDir)include\mono-2.0"
         Assemblies="@(_AndroidAssembliesInternal)"
         MainLibraryFileName="$(MainLibraryFileName)"
-        IncludeNetworkSecurityConfig="$(IncludeNetworkSecurityConfig)"
```

Comment:
The only purpose of the network security config was to trust our self-signed certificates in tests. Currently we don't have any use case for it. I think it would be better to remove the code for now and only bring it back if we find a new use case for it.

Reply:
ok